### PR TITLE
rollback with minor changes

### PR DESCRIPTION
**Pull request by ankitdas13**

### CODE REVIEW

File: src/Webhook.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function all($options = array())
      * @param string $id
      * @return Webhook
      */
-    public function edit($id, array $attributes = array())
+    public function edit($attributes, $id)
```
Comment (abdulwahidsharief) [inline src/Webhook.php:33]: @ankitdas13 can we maintain sequence ($id, $attributes)

Comment (enishant) [reply src/Webhook.php:33]: @abdulwahidsharief With this PR sequence is fixed, which was changed previously with commit https://github.com/razorpay/razorpay-php/commit/c673dfa71823e8a2ac452ec913c0d7a72fe6b3f3 at PR https://github.com/razorpay/razorpay-php/pull/329 

@ankitdas13 Please include `$attributes` parameter as `$attributes = array()`

Comment (ankitdas13) [reply src/Webhook.php]: @enishant i have added `$attributes` parameter as `$attributes = array()`
https://github.com/razorpay/razorpay-php/blob/webhook_edit_changes/src/Webhook.php#L33

**Review by enishant — APPROVED** [review]
LGTM

File: src/Webhook.php [hunk 2]
```diff
@@ -30,7 +30,7 @@ public function all($options = array())
      * @param string $id
      * @return Webhook
      */
-    public function edit($id, array $attributes = array())
+    public function edit($attributes = array(), $id)
     {
```
Comment (abdulwahidsharief) [inline src/Webhook.php:34]: @ankitdas13 attributes are by default empty array. please check this https://razorpay.slack.com/archives/C0132U6BQBY/p1663826045419419?thread_ts=1663822000.616219&cid=C0132U6BQBY

Comment (ankitdas13) [reply src/Webhook.php:34]: @enishant ^ should i remove default empty array .

Comment (enishant) [reply src/Webhook.php:34]: @ankitdas13 `$attributes` should not be empty array.

Comment (ankitdas13) [reply src/Webhook.php:34]: @enishant i have removed empty array is it ok now ?
https://github.com/razorpay/razorpay-php/blob/webhook_edit_changes/src/Webhook.php#L33

**Pull request merged**